src/main/java/emissary/core/IBaseDataObjectDiffHelper.java

```diff
@@ -226,6 +238,44 @@ public static void diff(final Map<String, byte[]> map1, final Map<String, byte[]
         }
     }
 
+    public static void keyValueMapDiff(final Map<String, Collection<String>> parameter1, final Map<String, Collection<String>> parameter2,
```

Comment:
Do you think it would be helpful to add javadoc comments for these two functions?